pystac/html/jinja_env.py

```diff
@@ -0,0 +1,10 @@
+jinja_env = None
+
+try:
+    from jinja2 import Environment, PackageLoader, select_autoescape
+
+    jinja_env = Environment(
```

Comment:
Good point. How about wrapping all of this in a cached function? Something like:

```python
from functools import lru_cache

@lru_cache()
def get_jinja_environment():
    try:
        from jinja2 import Environment, PackageLoader, select_autoescape

        return Environment(
            loader=PackageLoader("pystac", "html"), autoescape=select_autoescape()
        )
    except ModuleNotFoundError:
        return None
```

And then only importing and using the `get_jinja_environment` function?

Reply:
I'm +1 for using a cached function to retrieve this as-needed.

Reply:
I will go with the cached function approach. I will use `# type: ignore` since I can't use `Optional[Environment]` as the class hasn't been imported at function definition time.